docs/imagecustomizer/developer-guide.md

```diff
@@ -12,6 +12,9 @@ Run:
 sudo make -C ./toolkit go-imagecustomizer
 ```
 
+If you're updating the image customizer api as part of your change, do update
```

Comment:
I generally like to place commands to run in multiline codeblocks. IMHO, this makes it easier to find and copy the commands.